lib/rspec/matchers/built_in/raise_error.rb

```diff
@@ -64,7 +62,7 @@ def matches?(given_proc, negative_expectation=false, &block)
 
         # @private
         def does_not_match?(given_proc)
-          prevent_invalid_expectations
+          warn_about_invalid_expectations
```

Comment:
Appveyor needed a kick anyway and I didn't feel that strongly about it so changed it ;)